pipeline-qsub/process_sample.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -x
+
+if [ $# -ne 2 ]; then
+    echo "Usage: $0 WORK_DIR SAMPLE"
+    exit 1
+fi
+
+# Command line arguments
+WORK_DIR="$1"
+SAMPLE="$2"
+
+# Standard input file names
+DNABC_OUTPUT_DIR="${WORK_DIR}/dnabc_results"
+
+# Standard output file names
+SUMMARY_DIR="${WORK_DIR}/summary"
+ILLQC_SUMMARY="$SUMMARY_DIR/summary-illqc_${SAMPLE}.json"
+DECONTAM_SUMMARY="${SUMMARY_DIR}/summary-decontam_${SAMPLE}.json"
+PHYLO_SUMMARY="${SUMMARY_DIR}/summary-phylo_${SAMPLE}.json"
+PATHWAY_SUMMARY="${SUMMARY_DIR}/summary-pathway_${SAMPLE}.json"
+ILLQC_OUTPUT_DIR="illqc_results"
+ILLQC_QC_OUTPUT_DIR="illqc_reports"
+DECONTAM_OUTPUT_DIR="decontam_results"
+PHYLO_OUTPUT_DIR="phyloprofiler_results"
+PATHWAY_OUTPUT_DIR="pathfinder_results"
+
```

Comment:
Should we concatenate the WORK_DIR to the output folders of the pipeline steps? (e.g. ILLQC_OUTPUT_DIR=${WORK_DIR}/"illqc_results") Or are we assuming that we are moving the bash file where we want to populate the results?